zmon_worker_monitor/workflow.py

```diff
@@ -19,11 +19,12 @@
 import settings
 from zmon_worker_monitor import eventloghttp
 import snappy
+import opentracing as ot
 
 from redis_context_manager import RedisConnHandler
 from tasks import configure_tasks
 from tasks import check_and_notify, trial_run, cleanup
-
+from basictracer.text_propagator import TextPropagator
```

Comment:
This uses `ot-trace-*` as keys in the dict, see https://github.com/opentracing/basictracer-python/blob/master/basictracer/text_propagator.py#L7

depending on the implementation, different keys will be in use:
https://github.com/instana/golang-sensor/blob/master/propagation.go#L14 
... quoting opentracing python code:

> NOTE: The TEXT_MAP carrier dict may contain unrelated data (e.g.,
>     arbitrary gRPC metadata). As such, the Tracer implementation should use a
>     prefix or other convention to distinguish Tracer-specific key:value
>     pairs.

so ot.tracer.extract() must be used to support the implementation specific keys